### ISSUE TITLE
[Help] Retrieve the host / BaseURI / origin of request

### ISSUE THREAD

**Issue by silverbackdan** [nuxt-community/express-template#58]
I am wondering whether there is a way to get the host of the application with this setup. I think that in NuxtJS standalone you can call `this.origin` although I'm not sure now whether it returned the hostname when rendered client-side or server-side.

I tried looking at the baseURI of an element, but of course when rendered from the server, this isn't available.

Basically I'm just trying to create a simple function to determine a path to an API depending on if I'm in development or production. This is my mixin:
```js
let mixins = {
  methods: {
    apiUrl: function (src) {
      let baseURI = 'http://example.com'
      return (process.env.NODE_ENV !== 'production' ? (baseURI + ':9000') : baseURI.replace('//', '//api.')) + '/' + src
    }
  }
}

exports.mixin = mixins
```

So basically in development it'd output `http://developmentdomain.com:9000/path/to/call` and production would be `https://api.productiondomain.com/path/to/call`

I know the NODE_ENV is not available in development, but that's OK because it is in production so that bit works. But if I use HOST then it works on server-side calls, and this['$el'].baseURI would work on client-side but it seems a bit hacky and I'm hoping there's something simple I'm missing because I'm new to this.

Thanks in advance for any advice.

**Comment by silverbackdan** [comment]
Silly query really. I may as well hard-code the 2 URLs in. No effort really in doing that and I think it may even be possible with async data but I'm sure I'll see if that works as I continue playing around.
